app/serializers/api/v1/participant_outcome_serializer.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "jsonapi/serializer/instrumentation"
+
+module Api
+  module V1
+    class ParticipantOutcomeSerializer
+      include JSONAPI::Serializer
+      include JSONAPI::Serializer::Instrumentation
+
+      set_id :id
+      set_type :'participant-outcome'
+
+      attribute :completion_date do |outcome|
+        outcome.completion_date.rfc3339
+      end
+
+      attribute :course_identifier do |outcome|
+        outcome.participant_declaration.course_identifier
+      end
+
+      attribute :has_passed do |outcome|
+        outcome.state == "passed"
```

Comment:
Forgot about the state 😅 
Since we have a third state which is voided, should we match what we intend to surface on declarations: `true/false/nil` or maybe surface state as it is  with `pass,fail,voided` to avoid confusion?

Reply:
Addressed in https://github.com/DFE-Digital/early-careers-framework/pull/2782/commits/cd978be007b0e245245461e4fe6deb60b9943d91